source/installation-guide/installing-wazuh-manager/linux/amazon/wazuh_server_packages_amazon.rst

```diff
@@ -88,11 +71,11 @@ Installing the Wazuh API
       # service wazuh-api status
 
 .. note::
-    Now that the Wazuh API is installed, check out the section :ref:`securing_api` to set up some additional settings.
+    Check out the section :ref:`securing_api` to set up some additional settings.
```

Comment:
I would change the securing_api reference for the RESTful API index reference so the user can access any API related documentation. Something like:

Check out the section INDEX for more information on how to set up and use Wazuh API.